### DISCUSSION TITLE
Recommended EdgeDB workflow

### DISCUSSION THREAD

**Discussion by liegeandlief** [geldata/gel#4542] [Q&A]
Is there a recommended worflow for working on an EdgeDB project across a team? I would be insterested to know if there are any best practices for the following:

1. Setting up a development environment.
2. Ensuring that all team members are using the same development environment.
3. Ensuring that the development installation is of the same version and has the same configuration as the production installation.
4. Handling multiple simultaneous development branches and the potential conflicts which may arise from this.
5. EdgeDB's powerful access control, contraints and functions allow for a lot of business logic to live in an EdgeDB project. Is there a recommended way to unit test a project?

Thank you!

**Comment by haikyuu** [comment]
@liegeandlief try asking in discord, these are very nice questions I'd like answered as well so I copied your question there https://discord.com/channels/841451783728529451/1031827740371992597/1031827740371992597
